chapters/results/sections/performance/performance.tex

```diff
@@ -0,0 +1,29 @@
+\section{Performance}
```

Comment:
I don't like this section in this chapter

Reply:
This will probably depend on the thesis structure we choose. I'm leaving it here for the time being